src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -229,6 +230,15 @@ def app_teardown(
 @app.command("deploy", requires_connection=True)
 @with_project_definition("native_app")
 def app_deploy(
+    prune: Optional[bool] = typer.Option(
+        False,
```

Comment:
Shouldn't default be None so we can tell whether the user specified it explicitly? I'd expect this would be important when no files are provided (`deploy --no-prune` and `deploy` have different semantics)

Reply:
Yep, you're right! Fixed it